src/plugins/intel_gpu/tests/functional/single_layer_tests/dynamic/reorg_yolo.cpp

```diff
@@ -0,0 +1,99 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "shared_test_classes/single_layer/reorg_yolo.hpp"
+#include "shared_test_classes/base/ov_subgraph.hpp"
+#include "ie_precision.hpp"
+#include "ngraph_functions/builders.hpp"
```

Comment:
Is it possible to use some `openvino` include instead `ngraph`?

Reply:
remove unnecessary ngraph header declaration